docs: Added additional workaround steps for stalled requests (#10777)
vitejs/vite: docs/guide/troubleshooting.md

  $ sudo sysctl fs.inotify.max_user_instances=8192
  $ sudo sysctl fs.inotify.max_user_watches=524288
  ```

> #### Alternatively try Modifying Configs

Reviewer: Probably only the first word should start with a capital letter for consistency with the other sections. I'd suggest an alternate heading 
```suggestion
#### Modify `DefaultLimitNOFILE`
```

Reviewer: I'm not suggesting to make the new steps required - just questioning if they need their own heading. You could still start the text with "If the above steps don't work then..."

  $ sudo sysctl fs.inotify.max_user_watches=524288
  ```

> If the above steps don't work then you can also try modifying the following files.
> 
> - /etc/systemd/system.conf
> - /etc/systemd/user.conf
> 
> By adding in the following un-commented config: `DefaultLimitNOFILE=65536`. Note that these settings persist but a **restart is required**.
> 

Reviewer 2: ```suggestion
If the above steps don't work, you can try adding `DefaultLimitNOFILE=65536` as an un-commented config to the following files:

- /etc/systemd/system.conf
- /etc/systemd/user.conf

Note that these settings persist but a **restart is required**.

```
